### ISSUE TITLE
WebApp:az webapp config ssl import web app is required parameter

### ISSUE THREAD

**Issue by SamWhitby** [Azure/azure-cli#20983]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
running the command "az webapp config ssl import --key-vault example-kv --key-vault-certificate-name example-cert --resource-group exampe-rg" results in the error "(--name | --ids) are required"
![image](https://user-images.githubusercontent.com/3334161/149425153-42590210-da7b-4987-8059-2ece79001045.png)
However is is possible to import a certificate to a resource group without the need for an app service existing first via the api directly as seen in this terrafrom module https://github.com/hashicorp/terraform-provider-azurerm/blob/f2ac777d484662c83bb45ece1ca808e39357921b/vendor/github.com/Azure/azure-sdk-for-go/services/web/mgmt/2021-02-01/web/certificates.go#L82.

**To Reproduce**
Run "az webapp config ssl import --key-vault example-kv --key-vault-certificate-name example-cert --resource-group exampe-rg"

**Expected behavior**
App service name or id should not be required and the cert should be imported to the specified resource group and made available to all app services that are created in the future.

**Comment by yonzhan** [comment]
route to service team

**Comment by Kotasudhakarreddy** [comment]
@SamWhitby Thanks for your feedback! we will review and check the feasibility.
cc @panchagnula

**Comment by panchagnula** [comment]
@SamWhitby this is correct Certificates on App Service currently are not associated with a webapp but rather a webspace (note not RG) hence --name is not necessary
